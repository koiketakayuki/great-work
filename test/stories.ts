// tslint:disable-next-line:import-name
import initStoryshots from '@storybook/addon-storyshots';
initStoryshots({});
