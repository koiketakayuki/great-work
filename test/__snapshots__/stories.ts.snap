// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Welcome to test 1`] = `
<h1>
  GreatWork 
  Ryutaro
  !
</h1>
`;
